src/components/DetailsMetrics/DetailsMetrics.scss

```diff
@@ -0,0 +1,290 @@
+
+.metrics {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 16px;
+  height: fit-content;
+  margin-bottom: 30px;
+
+  &__app-name {
+    font-size: 20px;
+    font-weight: 700;
+    color: $primary;
+  }
+
+  &__card-tmp {
+    z-index: 1;
+    top: 0;
+    position: sticky;
+    box-sizing: border-box;
+    border: $primaryBorder;
+    border-radius: 12px;
+    background: $white;
+    padding-top: 12px;
+    width: 100%;
+    transition: 0.1s;
+    animation: ease-in-out;
+    max-height: 140px;
+  }
+
+  &__card {
+    border-radius: 12px;
+    border: $primaryBorder;
+    padding: 6px 16px 35px;
+    transition: ease-in-out;
+    width: 100%;
+
+    &-drift-status {
+      margin-left: 8px;
+      border-radius: 50%;
+      height: 10px;
+      width: 10px;
+      display: inline-block;
+    }
+
+    &-drift-status-possible_drift {
+      background-color: $grandis;
+    }
+
+    &-drift-status-drift_detected {
+      background-color: $ceriseRed;
+    }
+
+    &-drift-status-no_drift {
+      background-color: $brightTurquoise;
+    }
+
+    &-drift-no_status {
+      background-color: $doveGray;
+    }
+
+    &-header {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      align-items: center;
+      gap: 8px;
+      padding-left: 5px;
+      font-weight: 500;
+
+      &-result-kind {
+        display: flex;
+        width: max-content;
+        justify-content: center;
+        align-self: center;
+        gap: 4px;
+        overflow: hidden;
+        color: $topaz;
+        text-overflow: ellipsis;
+        font-family: Roboto;
+        font-size: 14px;
+        font-style: normal;
+        font-weight: 500;
+        line-height: normal;
+      }
+
+      &-data {
+        color: $topaz;
+        text-align: center;
+        leading-trim: both;
+        text-edge: cap;
+        font-weight: 700;
+        font-size: 18px;
+      }
+
+      &-label {
+        font-size: 14px;
+      }
+    }
+
+    &-body {
+      display: flex;
+      flex: 1;
+      align-self: stretch;
+      width: 100%;
+      aspect-ratio: 5;
+      gap: 20px;
+      align-items: stretch;
+      position: relative;
+      box-sizing: border-box;
+      transition: 0.3s;
+      animation: ease-in-out;
+
+      .chart-item {
+        flex: 1;
+
+        canvas {
+          width: 100%;
+          height: 100%;
+        }
+      }
+
+      &-bar {
+        flex: 0 1 25%;
+        min-width: 210px;
+
+        canvas {
+          width: 100% ;
+          height: 100% ;
+        }
+
+        canvas.hidden-canvas {
+          position: absolute;
+          opacity: 0;
+          left: 0;
+          top: 0;
+        }
+      }
+
+      &-line {
+        flex: 1 1 0;
+        margin-top: 4px;
+
+        canvas {
+          width: 100% !important;
+          height: 100% !important;
+        }
+
+        canvas.hidden-canvas {
+          position: absolute;
+          opacity: 0;
+          left: 0;
+          top: 0;
+        }
```

Comment:
you haven't moved these styles as I wrote previously and marked this comment as resolved. You must create a mixin

Reply:
this comment is unresolved.
i'm making list of all comments (includes old ones, will add them to gira task)